spec/factories.rb

```diff
@@ -0,0 +1,7 @@
+FactoryGirl.define do
+  factory :book do
+    title { Faker::Company.catch_phrase }
+    description { Faker::Hacker.say_something_smart }
+    cover_url { Faker::Avatar.image(nil, '100x200') }
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.